### ISSUE TITLE
May not need History anymore

### ISSUE THREAD

**Issue by forforf** [forforf/justgraphit#9]
History.(j|t)s was created to be able to access last used keys and if memory serves it was a way to handle global state. However with the rewrite to functional React, it may not be necessary anymore.

**Comment by forforf** [comment]
Changing AppRouter to the  below will probably eliminate the need for History:
```
function AppRouter({storage}) {
  // const lastObject = storage.getLastStorageObject();
  const initialObject = {[storage.getInitialKey()]: storage.load(storage.getInitialKey())};

  // state
  const [graphObject, setGraphObject] = useState(GraphObject.fromPlainObject(initialObject));
  const [graphNameList, setGraphNameList] = useState(storage.getAllKeys());
```

**Comment by forforf** [comment]
fixed in e94adec
